### DISCUSSION TITLE
error saat knit

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#451] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/201813618-0bde16ee-de0e-4723-8a84-1c6c64e5bb4a.png)
![image](https://user-images.githubusercontent.com/114885734/201813750-6ad4dfc5-cd1a-400a-9ea5-1578bdb9d2bc.png)

Halo saya mau tanya error pada saat knit ini kenapa ya? Di Rmd nya (saat chunk tersebut dijalankan) tidak masalah, tapi saat knit error. 
>> sori, ralat, ternyata Rmd nya eror jg saat dijalankan lagi

**Comment by kevbow** [comment, marked as answer]
Halo @19807311 ,

Apakah sudah dipastikan bahwa di bagian atas dari chunk ini Anda sudah mendefinisikan `bank_train_x`? Karena error yang dihasilkan adalah object/file tidak ditemukan.

**Reply by 19807311** [reply]
sudah ketemu mas. makasih
